### ISSUE TITLE
(Zenfone Max M1) (Pie) Bootloop 

### ISSUE THREAD

**Issue by rhudsonsamuel** [phhusson/treble_experimentations#762]
Hi,
I have a problem with Many GSI's...
Many Gsi's are not working for me..So, I collected the log...
kindly help me... 
Whenever I install GSI it results in a boot loop. I  tried to collect logs but many ended in failure..But one GSI permitted me to take logs. My device forum is down because there is no developer for this Device.

**Device Specifications**
**Asus Zenfone Max M1 (SD430)** (PIE)

[log12.txt](https://github.com/phhusson/treble_experimentations/files/3706061/log12.txt)

**Comment by phhusson** [comment]
I'm only interested in my own GSIs.
Does my pie Aosp gsi work?

Le mer. 9 oct. 2019 à 09:29, Hudson Samuel <notifications@github.com> a
écrit :

**Comment by rhudsonsamuel** [comment]
yeah, worked sorry to ask here...but i taught you can help me...so i posted here

**Comment by murdika** [comment]
meybe your vendor doesnt support for other gsi,,u can downgrade using previuos stokrom,ask on group from tele or facebook,
